internal/runner/accept.go

```diff
@@ -325,9 +328,7 @@ func (r *Runner) prepareAndExecuteJob(
 		if err == nil {
 			// Execute the job. We have to close the UI right afterwards to
 			// ensure that no more output is writting to the client.
-			log.Info("starting job execution")
```

Comment:
I removed these log lines cause they're duplicated (the func that calls this does this same thing)